packages/core/src/shared/utilities/download.ts

```diff
@@ -14,6 +14,7 @@ import fs from '../fs/fs'
 export class ExportResultArchiveStructure {
     static readonly PathToSummary = path.join('summary', 'summary.md')
     static readonly PathToDiffPatch = path.join('patch', 'diff.patch')
+    static readonly PathToMetrics = path.join('metrics', 'metrics.json')
```

Comment:
join() is not needed here (even on Windows).
```suggestion
    static readonly PathToMetrics = 'metrics/metrics.json'
```